.vib/couchdb/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - couchdb
+  - couchjs
+  - erl
+  - gosu
+  - ini-file
```

Comment:
I think you might be missing `wait-for-port`, as the Dockerfile indicates:

https://github.com/bitnami/containers/blob/53225801ec429d4bae24ea3b2b10458e45267398/bitnami/couchdb/3/debian-11/Dockerfile#L25